presto-main/src/main/java/com/facebook/presto/operator/WindowInfo.java

```diff
@@ -119,7 +119,7 @@ public synchronized DriverWindowInfo build()
     }
 
     @Immutable
-    private static class DriverWindowInfo
+    public static class DriverWindowInfo
```

Comment:
I made it here because previously there was no need to make it public, until now.
I can move it there, if you feel it's important.